open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/VirtualizationConfig.java

```diff
@@ -136,37 +141,54 @@ public void setIvOutTopic(Connection ivOutTopic) {
         this.ivOutTopic = ivOutTopic;
     }
 
+    /**
+     * Provide the connection of the virtualization solution
+     * @return Connection virtualizationSolution
+     */
+    public Connection getVirtualizationSolution() {
+        return virtualizationSolution;
+    }
 
     /**
-     * Validate that an object is equal depending on their stored values.
-     *
-     * @param objectToCompare object
-     * @return boolean result
+     * Set the connection of the information view out topic
+     * @param virtualizationSolution Connection
      */
-    @Override
-    public boolean equals(Object objectToCompare) {
-        if (this == objectToCompare) {
-            return true;
-        }
-        if (objectToCompare == null || getClass() != objectToCompare.getClass()) {
-            return false;
-        }
-        VirtualizationConfig that = (VirtualizationConfig) objectToCompare;
-        return  Objects.equals(getIvInTopicName(), that.getIvInTopicName())&&
-                Objects.equals(getIvInTopic(), that.getIvInTopic()) &&
-                Objects.equals(getIvOutTopicName(), that.getIvOutTopicName()) &&
-                Objects.equals(getVirtualizationProvider(), that.getVirtualizationProvider());
+    public void setVirtualizationSolution(Connection virtualizationSolution) {
+        this.virtualizationSolution = virtualizationSolution;
     }
 
+    /**
+     * Provide the connection of the virtualization configuration
+     * @return virtualizationSolutionConfig
+     */
+    public Map<String, Object> getVirtualizationSolutionConfig() {
```

Comment:
Map<String, Object> is not a Connection object

Reply:
Hi Mandy, this Map object is used to extract settings from JSON in the rest call

{
	"class": "VirtualizationConfig",
	"virtualizationProvider": "org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.derby",
	"ivInTopicName": "InTopic",
	"ivOutTopicName": "OutTopic",
	"virtualizationSolutionConfig": {
		"serverAddress": "jdbc:derby://127.0.0.1:6414/gaiandb",
		"databaseName": "ING",
		"schema": "JSON",
		"username": "gaiandb",
		"password": "passw0rd",
		"timeoutInSecond": 5,
		"create": true,
		"derbyDriver": "org.apache.derby.jdbc.ClientDriver",
		"gdbNode": "GDB_NODE",
		"logicTableName": "TEST",
		"logicTableDefinition": "LTDE",
		"getLogicTables": "call listlts()"
	}
}

Reply:
So the comment needs updating ...